Assets/Mirror/Runtime/NetworkClient.cs

```diff
@@ -95,6 +144,15 @@ public virtual void Disconnect()
             }
         }
 
+        void RemoveTransportHandlers()
```

Comment:
Are you calling this anywhere?   I think you forgot that part

Reply:
fixed